engine/runtime/eval-expr/src/module/str.rs

```diff
@@ -22,6 +22,37 @@ pub(crate) mod str_module {
         let regex = Regex::new(regex).unwrap();
         regex.is_match(haystack)
     }
+
+    pub fn string_to_int(s: String) -> i64 {
+        s.parse::<i64>().unwrap()
+    }
+
+    pub fn string_to_f64(s: String) -> f64 {
+        s.parse::<f64>().unwrap()
```

Comment:
What does the error look like when this fails?
Isn't it better to use `expect` and provide a clear error message?